src/script/descriptor.cpp

```diff
@@ -1098,7 +1098,7 @@ std::string GetDescriptorChecksum(const std::string& descriptor)
 {
     std::string ret;
     std::string error;
-    Span<const char> sp(descriptor.data(), descriptor.size());
+    Span<const char> sp{descriptor};
```

Comment:
nit: could pass `descriptor` directly to `CheckChecksum`

Reply:
That doesn't work; `CheckChecksum` needs a reference to a mutable `Span<const char>` (it's an input/output argument). You can't bind a temporary to that.